### PR TITLE
breaking news design tweaks

### PR DESCRIPTION
**Pull request by sndrs**
from 

screen shot 2015-01-14 at 14 01 44
screen shot 2015-01-14 at 14 01 31

to

screen shot 2015-01-14 at 13 59 04
screen shot 2015-01-14 at 13 59 19

### CODE REVIEW

File: static/src/stylesheets/module/onward/_breaking-news.scss [hunk 1]
```diff
@@ -72,7 +68,7 @@ $breaking-news-icon-width: 36px;
 
 .breaking-news__item-content {
     @include box-sizing(border-box);
-    padding: $gs-baseline 0 $gs-baseline ($gs-gutter + $breaking-news-icon-width);
+    padding: $gs-baseline * .5 0 $gs-baseline ($gs-gutter + $breaking-news-icon-width);
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/onward/_breaking-news.scss]: Might be worth bracketing this up for legibility. Seeing as you do it later on the line.

**Comment by sammorrisdesign** [comment]
:+1: Except for comments, if you want.

**Comment by paulrobertlloyd** [comment]
Ha ha, told you @sndrs!

**Comment by paulrobertlloyd** [comment]
Also, is that close icon actually a circle? Looks squashed…

**Comment by sndrs** [comment]
@paulrobertlloyd i might raise it a design glitch ;)

File: static/src/stylesheets/module/onward/_breaking-news.scss [hunk 2]
```diff
@@ -91,6 +87,7 @@ $breaking-news-icon-width: 36px;
 .breaking-news__item-kicker,
 .breaking-news__item-headline {
     @include fs-headline(2);
+    line-height: 1.3;
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/onward/_breaking-news.scss]: Where's me unit?

**Comment by paperboyo** [comment]
Hmm,

Not that you should care (you shouldn't!), but im my humblest of opinions the thin bar (from desktop BP?) looks better before than after (vertically centered). And it's more consistent with how other dismissible bar we use looks:
bar

Regards
Mateusz

**Comment by phamann** [comment]
:+1: Ship it|

File: static/src/stylesheets/module/onward/_breaking-news.scss (hunk 2 above)
Comment (sndrs) [inline static/src/stylesheets/module/onward/_breaking-news.scss]: no one uses units anymore

**Pull request merged**